clipper_admin/clipper_admin/deployers/tensorflow.py

```diff
@@ -196,9 +196,11 @@ def predict(sess, inputs):
                     raise e
             else:
                 # Check if all the 4 files are present
-                if glob.glob('data/*.meta') and glob.glob(
-                        'data/*.index') and glob.glob(
-                            "data/checkpoint") and glob.glob("data/*.data*"):
+                if glob.glob(tf_sess_or_saved_model_path + "/*.meta") and glob.glob(
+                    tf_sess_or_saved_model_path + "/*.index") and glob.glob(
+                    tf_sess_or_saved_model_path + "/checkpoint") and glob.glob(
+                    tf_sess_or_saved_model_path + "/*.data*"):
```

Comment:
Instead of doing string concatenation, can you use `os.path.join` (e.g `os.path.join(tf_sess_or_saved_model_path, *.meta")`)